js/storage.js

```diff
@@ -0,0 +1,31 @@
+
+const username = document.getElementById('username');
+const email = document.getElementById('email');
+const textBox = document.getElementById('message');
+
+function storeData() {
+    const formData = {
+        username: username.value,
+        email: email.value,
+        message: textBox.value,
+    };
+
+    localStorage.setItem('formData', JSON.stringify(formData));
+}
+
+function getData() {
+    if (!localStorage.getItem('formData')) {
+        storeData();
+    }
+
+    const localData = JSON.parse(localStorage.getItem('formData'));
+    username.setAttribute('value', localData.username);
+    email.setAttribute('value', localData.email);
+    textBox.textContent = localData.message;
+}
+
```

Comment:
- [ ] You didn't export this file and Import it to the main file so it is not working. Kindly check this and make changes. You can check this by commenting (<script src="js/script.js"></script>
    <script src="js/newmodal.js"></script>
    <script src="js/contact-form.js"></script>) this code in HTML to see whether the local storage is working or not.